Demo/Demo/Models/PaymentTokenRequest.swift

```diff
@@ -0,0 +1,31 @@
+import Foundation
+import CorePayments
+
+struct PaymentTokenRequest {
+    
+	let setupToken: String
+    
+    var path: String {
+        return "/payment_tokens/"
+    }
+    
+    var method: CorePayments.HTTPMethod {
+        return .post
+    }
+    
+    var headers: [CorePayments.HTTPHeader: String] {
+        return [.contentType: "application/json"]
```

Comment:
Does this API request need us to send the base64 encoded client token in the header?

Reply:
It's handled in the sample merchant server. Sends full access token.